e2e-tests/src/staking.rs

```diff
@@ -97,3 +100,107 @@ pub fn wait_for_era_completion(
     wait_for_session(connection, first_session_in_next_era)?;
     Ok(next_era_index)
 }
+
+pub fn check_non_zero_payouts_for_era(
+    node: &String,
+    stash: &KeyPair,
+    connection: &Connection,
+    era: BlockNumber,
+) {
+    let stash_account = AccountId::from(stash.public());
+    let locked_stash_balance_before_payout = locks(&connection, &stash);
+    assert!(
+        locked_stash_balance_before_payout.is_some(),
+        "Expected non-empty locked balances for account {}!",
+        stash_account
+    );
+    let locked_stash_balance_before_payout = locked_stash_balance_before_payout.unwrap();
+    assert_eq!(
+        locked_stash_balance_before_payout.len(),
+        1,
+        "Expected locked balances for account {} to have exactly one entry!",
+        stash_account
+    );
+    payout_stakers(node, stash.clone(), era - 1);
+    let locked_stash_balance_after_payout = locks(&connection, &stash);
+    assert!(
+        locked_stash_balance_after_payout.is_some(),
+        "Expected non-empty locked balances for account {}!",
+        stash_account
+    );
+    let locked_stash_balance_after_payout = locked_stash_balance_after_payout.unwrap();
```

Comment:
okay, the above comment seems to be more general :)

Reply:
Will fix